src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -292,6 +302,47 @@ private bool IsManifestIdOutdated(string workloadManifestDir)
             return _outdatedManifestIds.Contains(manifestId);
         }
 
+        private Dictionary<string, WorkloadSet> GetAvailableWorkloadSets()
+        {
+            Dictionary<string, WorkloadSet> availableWorkloadSets = new Dictionary<string, WorkloadSet>();
+
+            foreach (var manifestRoot in _manifestRoots.Reverse())
```

Comment:
Are we doing the .Reverse so that the dotnet directory goes first before the user home directory when picking the newest available workload set?

It looks like we do that above as well. May we just re-order the manifestRoots?